app/views/decidim/half_signup/quick_auth/verify.html.erb

```diff
@@ -21,7 +21,7 @@
                 <div id="verification" class="flex-digits">
                   <% auth_code_length.times do |ind| %>
                     <div class="flex-digits-input">
-                      <input type="text" pattern="[0-9]*" name="<%= "digit#{ind + 1}" %>" autocomplete="off" aria-label="<%= t(".inputs", count: ind + 1) %>">
+                      <input type="number" pattern="[0-9]" name="<%= "digit#{ind + 1}" %>" autocomplete="off" aria-label="<%= t(".inputs", count: ind + 1) %>">
```

Comment:
```suggestion
                      <input type="number" pattern="[0-9]*" name="<%= "digit#{ind + 1}" %>" autocomplete="off" aria-label="<%= t(".inputs", count: ind + 1) %>">
```

The `*` allow iPhones to display automatically the numerical keyboard, is it needed to remove it ?

Reply:
Damn here is an other difference between Iphone and Android then because it didn't seem to be necessary on android, I don't actually think there will be a lot of changes by adding that if you say that it will fix one issue you just found !